src/kryptos/cli/main.py

```
@@ -302,7 +303,9 @@ def build_parser() -> argparse.ArgumentParser:
 
     sp_serve = sub.add_parser("serve", help="Run the Kryptos FastAPI server (turbovec RAG search over artifacts/)")
     sp_serve.add_argument("--host", type=str, default="0.0.0.0", help="Bind host (default: 0.0.0.0)")
-    sp_serve.add_argument("--port", type=int, default=8000, help="Bind port (default: 8000)")
+    sp_serve.add_argument(
+        "--port", type=int, default=int(os.environ.get("PORT", "8000")), help="Bind port (default: $PORT or 8000)"
```

Pylint (4.0.7)

[convention] Line too long (114/88)

(C0301)